doc/content/source/materials/SteelCreepDamageOh.md

```diff
@@ -0,0 +1,45 @@
+# Steel Creep Damage (Oh et al)
+
+!syntax description /Materials/SteelCreepDamageOh
+
+## Description
+
+The `SteelCreepDamageOh` can model the mechanical failure of steel due to creep (see [!cite](oh2011creep) and [!cite](OH2011124)).
+
+Accumulated damage ($\omega$) is computed by accumulating the creep strain rate normalized with the multi-axial creep ductility. The accumulated damage is 
+\begin{equation}
+\omega=\sum{\Delta\omega = \sum{\frac{\Delta\varepsilon_{x}}{\varepsilon^{*}_{f}}}}
+\end{equation}
+
+The triaxial creep ductility $\varepsilon^{*}_{f}$ can be obtained from the uniaxial creep ductility $\varepsilon_f$ using the following expression
+
+\begin{equation}
+\varepsilon^{*}_{f} = \varepsilon_{f} \frac{\sinh{\frac{2.0}{3.0} \frac{n - 0.5}{n + 0.5}}}{\sinh{2 \frac{n - 0.5}{n + 0.5} h}}
+\end{equation}
+
+where $n$ is the creep power law exponent, $h$ is the stress triaxiality.
+
+!alert note
+The accumulated damage $\omega$, as defined in [!cite](OH2011124), can be requested by the user as an auxiliary variable. In addition, the more traditional `damage_index`, which represents a linear change in the stress value, can also be requested. This class internally casts the value of $\omega$ into the `damage_index`.
```

Comment:
I think it's worth making it clear that they omega and damage_index are not identical -- that damage_index is zero until the threshold value of omega is reached.